tests/testthat/test-utils_review.R

```diff
@@ -0,0 +1,83 @@
+test_that("md_to_word generates expected Word doc", {
+  docx_file <- "review.docx"
+  md_dir <- local_create_md()
+  rv_dir <- gsub("markdown", "review", md_dir)
+  docx_path <- local_create_word_doc(
+    md_dir = md_dir,
+    rv_dir = rv_dir,
+    docx_file = docx_file
+  )
+
+  # When run for first time, just save the generated Word doc for future use
+  # Note the test will essentially be comparing this doc to itself on that first
+  # run!
+  if (!file.exists(docx_file)) file.copy(docx_path, docx_file)
+
+  # Get docx objects for comparisons
+  old <- officer::read_docx(docx_file)
+  new <- officer::read_docx(docx_path)
+
+  comp_structure <- waldo::compare(new, old)
+
+  # Due to how officer::read_docx works, we expect there to always be
+  # differences in the package_dir and doc_properties$data.
+  # Also, when run on github CI, there are small differences in some of the
+  # colours used in the Word doc styles.
+  expected_differences <- ifelse(testthat:::on_ci(), 8, 2)
```

Comment:
@kygoffe I misremembered how I ended up handling the test on CI and locally. The code above explains :) I've just moved the `is_ci()` call inline since it is used once. So we expect 2 differences locally, and 8 differences on CI.